cmd/kubeadm/app/cmd/cmd.go

```diff
@@ -89,7 +89,7 @@ func NewKubeadmCommand(in io.Reader, out, err io.Writer) *cobra.Command {
 	cmds.AddCommand(NewCmdVersion(out))
 	cmds.AddCommand(NewCmdToken(out, err))
 	cmds.AddCommand(upgrade.NewCmdUpgrade(out))
-	cmds.AddCommand(alpha.NewCmdAlpha(out))
+	cmds.AddCommand(alpha.NewCmdAlpha(in,out))
```

Comment:
Fix formatting to get test passing `cmds.AddCommand(alpha.NewCmdAlpha(in, out))` -> add a space between `in,` and `out`